lms/templates/problem.html

```diff
@@ -68,6 +68,7 @@ <h3 class="hd hd-3 problem-header" id="${ short_id }-problem-title" aria-describ
           </span>
           <span class="sr">(${submit_disabled_cta['description']})</span>
         % else:
+            % if submit_disabled_cta.get('link'):
             <form class="submit-cta" method="post" action="${submit_disabled_cta['link']}">
```

Comment:
Nit: would it make sense to use `get` here and add the `if submit_disabled_cta.get('link'):` condition lower (i.e., to hide only the input and button tags) to avoid code repetition?